.npmrc

```diff
@@ -1,2 +1,3 @@
 ; adding this as npm 7 automatically installs peer dependencies but pnpm does not
-auto-install-peers=true
\ No newline at end of file
+auto-install-peers=true
+strict-peer-dependencies=false
```

Comment:
Sadly I think this is a must to have right now. I tried fixing peer deps but it was a huge problem to try solve. Warnings show up in CI and in pnpm i and I hope these will prompt teams to fix peer deps.